client-extensions/liferay-object-collection-filter-extension/src/common/services/objectDefinition.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
```

Comment:
you can reference Loc [PR](https://github.com/tototrinh/lfr-client-extensions/pull/6/files) for implement service file